### DISCUSSION TITLE
Anyway to disable fuse?

### DISCUSSION THREAD

**Discussion by groogiam** [containers/buildah#5582] [Q&A] [answered]
I run a container inside azure container instances that uses buildah to build and push other containers on demand. This all worked in version 1.19 but it appears in later version that build bud now relies on /dev/fuse being passed in from the host. Is there anyway to bypass this and use the old method that did not rely on this device? The dependency on this device seems to break the ability of buildah to run in a container where you don't control the host machine. Thanks.

**Comment by groogiam** [comment, marked as answer]
Looks like `--storage-driver vfs` can be used to force VFS to be used rather than fuse.

**Comment by rhatdan** [comment]
Fuse is required for systems that do not support native overlay in rootless containers. vfs as you found out will work regardless, although may be slower.
